test/harry/main/org/apache/cassandra/harry/sut/injvm/InJvmSutBase.java

```diff
@@ -45,30 +46,42 @@
 import org.apache.cassandra.distributed.api.IInstanceConfig;
 import org.apache.cassandra.distributed.api.IMessage;
 import org.apache.cassandra.distributed.api.IMessageFilters;
+import org.apache.cassandra.exceptions.RequestTimeoutException;
 import org.apache.cassandra.harry.core.Configuration;
 import org.apache.cassandra.harry.sut.SystemUnderTest;
+import org.apache.cassandra.utils.AssertionUtils;
+import org.assertj.core.api.Condition;
 
 public class InJvmSutBase<NODE extends IInstance, CLUSTER extends ICluster<NODE>> implements SystemUnderTest.FaultInjectingSut
 {
     private static final Logger logger = LoggerFactory.getLogger(InJvmSutBase.class);
 
+    protected static final Condition<Object> TIMEOUT_CHECKER = AssertionUtils.isInstanceof(RequestTimeoutException.class);
```

Comment:
this is a utility class we use in accord to deal with the C* exceptions we expect, but the fact jvm-dtest throws with a different ClassLoader so type checks don't work... this uses string name + walking the input definition to find matches...